packages/venia-ui/lib/components/Checkout/addressForm.js

```diff
@@ -179,3 +178,61 @@ AddressForm.propTypes = {
 };
 
 export default AddressForm;
+
+export const SET_GUEST_EMAIL_MUTATION = gql`
```

Comment:
For deprecated components like this one (old drawer checkout), I just inlined the operations. It didn't make sense to me to decompose into talon and migrate operations.